tests/test_markduplicates_from_bam.yml

```diff
@@ -12,10 +12,6 @@
     - path: results/csv/recalibrated.csv
       md5sum: 1888a924bc70bd80165a96ad641e22d6
     - path: results/multiqc
-    - path: results/preprocessing/converted/test/test.converted.cram
-    # binary changes md5sums on reruns.
-    - path: results/preprocessing/converted/test/test.converted.cram.crai
-    # binary changes md5sums on reruns.
```

Comment:
where did they go?

Reply:
I'm actually not sure, I think they disappeared when I refactor that part in sarek.nf